Assets/scripts/leaderboard.cs

```diff
@@ -0,0 +1,84 @@
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using Shitty.Networking;
+
+public class leaderboard : MonoBehaviour
+{
+
+    public Text leaderboard_text;
+
+    public List<Score> scores = new List<Score>();
+
+
+    public leaderboard()
+    {
+
+    }
+
+    public void Start()
+    {
+        StartCoroutine(getLeaderboard());
+    }
+
+
+    public IEnumerator getLeaderboard()
+    {
+        Http http = new Http();
+        yield return StartCoroutine(http.Get("/scores"));
```

Comment:
@maxtondelaney this is how promises look in c#. 

notice they don't return anything. Classes must fetch and hold their own data. :vomit: